packages/polling-controller/src/StaticIntervalPollingController.ts

```diff
@@ -0,0 +1,86 @@
+import { BaseController, BaseControllerV1 } from '@metamask/base-controller';
+import type { NetworkClientId } from '@metamask/network-controller';
+import type { Json } from '@metamask/utils';
+
+import {
+  AbstractPollingControllerBaseMixin,
+  getKey,
+} from './AbstractPollingController';
+import type {
+  IPollingController,
+  PollingTokenSetId,
+} from './AbstractPollingController';
+
+type Constructor = new (...args: any[]) => object;
+
+/**
+ * StaticIntervalPollingControllerMixin
+ * A polling controller that polls on a static interval.
+ *
+ * @param Base - The base class to mix onto.
+ * @returns The composed class.
+ */
+function StaticIntervalPollingControllerMixin<TBase extends Constructor>(
+  Base: TBase,
+) {
+  abstract class StaticIntervalPollingController
+    extends AbstractPollingControllerBaseMixin(Base)
+    implements IPollingController
+  {
+    readonly #intervalIds: Record<PollingTokenSetId, NodeJS.Timeout> = {};
+
+    intervalLength: number | undefined = 1000;
```

Comment:
Since this property is currently public, you have two ways of accessing this property. This seems unnecessary. Thoughts on using `#intervalLength` instead of `intervalLength`?

Reply:
done here: https://github.com/MetaMask/core/pull/3636/commits/823aaa188f572a8aa76973a012ae9bfa8da518e0